examples/gke/xfs-formatter-daemonset.yaml

```diff
@@ -0,0 +1,77 @@
+# Daemonset that converts ephemeral-storage to xfs
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: xfs-formatter
+spec:
+  selector:
+    matchLabels:
+      name: xfs-formatter
+  template:
+    metadata:
+      labels:
+        name: xfs-formatter
+    spec:
+      hostPID: true
+      nodeSelector:
+        scylla.scylladb.com/gke-ephemeral-storage-local-ssd: "true"
+      tolerations:
+      - key: role
+        operator: Equal
+        value: scylla-clusters
+        effect: NoSchedule
+      containers:
+      - name: xfs-formatter
+        image: docker.io/scylladb/scylla-operator:latest
+        securityContext:
+          privileged: true
+        command:
+        - "/bin/bash"
+        - "-c"
+        args:
+        - |
+          set -euExo pipefail
+          shopt -s inherit_errexit
+
+          cp -r /usr/local/lib/scylla-operator/gke/xfs-formatter /host/var/lib/
+
+          if [[ ! -d /host/usr/local/lib/systemd ]]; then mkdir /host/usr/local/lib/systemd; fi
+          if [[ ! -d /host/usr/local/lib/systemd/system ]]; then mkdir /host/usr/local/lib/systemd/system; fi
+          cp -r /usr/local/lib/scylla-operator/gke/systemd/. /host/usr/local/lib/systemd/system
+
+          if [[ ! -d /host/usr/local/lib/systemd/system/kube-node-configuration.service.d ]]; then mkdir /host/usr/local/lib/systemd/system/kube-node-configuration.service.d; fi
+          cat<<-EOF > /host/usr/local/lib/systemd/system/kube-node-configuration.service.d/10-path.conf
+            [Service]
+            Environment="PATH=/var/lib/xfs-formatter/bin:$( chroot /host /bin/bash -c 'echo "${PATH}"' )"
+          EOF
+
+          if [[ ! -d /host/mnt/raid-disks ]]; then mkdir /host/mnt/raid-disks; fi
```

Comment:
(last time we chatted on this I forgot that you'll also need to handle reentrancy with ifs for the mkdir, otherwise I could have lived with `-p` even with the disadvantages, sorry)